modules/ROOT/pages/message-logging-policy.adoc

```diff
@@ -3,53 +3,55 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-This policy logs a custom message with the information available between policies, proxy, or backend at any point of the execution. This policy works with both log4j and log4j2.
-
-== Prerequisites
-
-You need to have the Anypoint Platform organization administrator role or have permission to create or manage APIs.
-
-== About Logging Policy Parameters
-
-The Message Logging Policy consists of these configuration parameters:
-
-image::apim-message-logging-policy-ui.png[Message Logging Policy UI]
-
-[%header%autowidth.spread]
+[width="100%", cols="5,15"]
 |===
-|Parameter |Description
-|Message |A DataWeave expression that's transformed into a message to be logged.
-
-Note: If the payload is used as part of the message log, do not configure the Listener as non-repeatable (by default this setting is off).
-|Conditional |The condition expressed as a DataWeave expression that returns true or false that needs to be satisfied to log the message. This field is optional.
-|Category |The category that the log has. This parameter is optional. If no category is set, the default is the `org.mule.runtime.logging.policy-<policy id>` value.
-|Level |Defines if the message is logged according to the configuration in your log4j2 file.
-|Before Calling API |Set to cause your logging policy to be applied before the API is called.
-|After Calling API |Set to cause your logging policy to be applied after the API is called.
+>s| Policy Name | Message Logging
+>s|Summary      | Logs messages user-password defined on LDAP
+>s|Category | Security
+>s|First Mule version available | 4.0.0
+.1+>.^s| Returned Status Codes
+|401 - Unauthorized or invalid client application credentials
+// what is the right ret code for this policy
 |===
 
-== About the Logging Format
-
-A log has the following format: `<date> <thread name> <level> <category>: <message>`
+This policy logs a custom message with the information available between policies, proxy, or backend at any point of the execution. This policy works with both Apache log4j and Apache log4j2.
 
-category org.mule and message #[payload] (where payload value is "the payload") is:
+//Nahuel--what is meant by information available between policies, proxy, or backend at any point of the execution
```

Comment:
Proxy communicates with a backend (unless the backend itself is the one to be tracked). Policies can be applied to a proxy in a layered (ordered) way. This is a policy so it can be  applied at certain point among policies, by using the "order" feature. So, this policy has the information available from the incoming request and policies currently applied before it, and/or information coming from the backend, and any policies currently applied between it  and the message policy.

Reply:
Done! Thanks!